README.md

```diff
@@ -1 +1,37 @@
-# ebx-linkedin-sdk
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ebx/ebx-linkedin-sdk.svg?branch=master)](https://travis-ci.org/ebx/ebx-linkedin-sdk)
+
+# ebx-linkedin-sdk
+
+## What it is
+
+ebx-linkedin-sdk is a pure Java LinkedIn API client with no external dependencies.
```

Comment:
It does have some external dependencies i.e. lombok, apache, org.json... unless you mean something else?

Reply:
yes I just copied this from RestFB's readme, I think it is wrong in their case too...will update!